src/main/java/com/lpvs/entity/dashboard/Grade.java

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Please move this file into `enum` folder.

Reply:
Done.